### PR TITLE
feat(GO): gameobject spawn addon editor

### PR DESCRIPTION
**Pull request by FrancescoBorzi**

### CODE REVIEW

File: src/app/config/app-routing.module.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ import { MailLootTemplateComponent } from '../features/other-loots/mail-loot/mai
 import { MailLootHandlerService } from '../features/other-loots/mail-loot/mail-loot-handler.service';
 import { SelectSpellComponent } from '../features/spell/select-spell/select-spell.component';
 import { SpellDbcComponent } from '../features/spell/spell-dbc/spell-dbc.component';
+import { GameobjectSpawnAddonComponent } from '../features/gameobject/gameobject-spawn-addon/gameobject-spawn-addon.component';
```
Comment (Kitzunu) [inline src/app/config/app-routing.module.ts:83]: Better use GameobjectAddon instead of "spawn" to match DB name imo

Comment (FrancescoBorzi) [reply src/app/config/app-routing.module.ts:83]: I only made this exception with `creature`, `gameobject` and their `_addon` table in order to make extra clear that it's about spawns and not the creature/gameobject itself. So this follows the same pattern for consistency :)

Comment (Kitzunu) [reply src/app/config/app-routing.module.ts:83]: True, but for not "spawn" you would have GameobjectTemplateAddon, no?

Comment (FrancescoBorzi) [reply src/app/config/app-routing.module.ts:83]: yes

Comment (Kitzunu) [reply src/app/config/app-routing.module.ts:83]: is whatever I guess

**Review by Helias — APPROVED** [review]

**Pull request merged**